units/src/locktime/absolute.rs

```diff
@@ -205,12 +216,20 @@ impl MedianTimePast {
     }
 
     /// Converts this [`MedianTimePast`] to a raw `u32` value.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use bitcoin_units::locktime::absolute;
+    ///
+    /// assert_eq!(absolute::MedianTimepast::MIN.to_u32(), 500_000_000);
```

Comment:
```suggestion
    /// assert_eq!(absolute::MedianTimePast::MIN.to_u32(), 500_000_000);
```

Typo causing the CI fail. Otherwise all looks good.